### PR TITLE
Fixed sitemap xml generation to improve performance

### PR DESCRIPTION
**Pull request by wachterjohannes**
__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| Related PRs      | none
| BC breaks        | none 
| Documentation PR | none

### CODE REVIEW

**Comment by chirimoya** [comment]
Check published and state for each language.

File: tests/Sulu/Component/Webspace/Loader/XmlFileLoaderTest.php [hunk 1]
```diff
@@ -306,17 +306,6 @@ public function testLoadWithNotExistingDefault()
         );
     }
 
-    public function testLoadWithoutDefaultLocalization()
```
Comment (wachterjohannes) [inline tests/Sulu/Component/Webspace/Loader/XmlFileLoaderTest.php]: i have changed this. if no default is selected (with xml) first is default. therfor this testcase will fail and i have removed it!

**Comment by wachterjohannes** [comment]
@chirimoya fixed also the open ghost bug! now it should be ready to review

**Comment by chirimoya** [comment]
ack

**Pull request merged**